### PR TITLE
Question Answering support for Albert and Roberta in TF

### PR DESCRIPTION
**Pull request by Pierrci**
This PR simply adds `TFRobertaForQuestionAnswering` and `TFAlbertForQuestionAnswering` classes (I needed them to do some model conversions!)

### CODE REVIEW

File: src/transformers/modeling_tf_auto.py [hunk 1]
```diff
@@ -849,14 +855,16 @@ def from_config(cls, config):
...
+                    - isInstance of `roberta` configuration class: RobertaModel (RoBERTa model)
                     - isInstance of `bert` configuration class: BertModel (Bert model)
                     - isInstance of `xlnet` configuration class: XLNetModel (XLNet model)
                     - isInstance of `xlm` configuration class: XLMModel (XLM model)
 
         Examples::
 
             config = BertConfig.from_pretrained('bert-base-uncased')    # Download configuration from S3 and cache.
-            model = AutoModelForSequenceClassification.from_config(config)  # E.g. model was saved using `save_pretrained('./test/saved_model/')`
+            model = TFAutoModelForQuestionAnswering.from_config(config)  # E.g. model was saved using `save_pretrained('./test/saved_model/')`
```
Comment (thomwolf) [inline src/transformers/modeling_tf_auto.py:867]: Good catch!

**Review by thomwolf — APPROVED** [review]
Nice and clean, thanks a lot @Pierrci

File: src/transformers/modeling_tf_albert.py [hunk 2]
```diff
@@ -826,3 +826,68 @@ def call(self, inputs, **kwargs):
...
         return outputs  # logits, (hidden_states), (attentions)
+
+
+@add_start_docstrings(
+    """Albert Model with a span classification head on top for extractive question-answering tasks like SQuAD (a linear layers on top of the hidden-states output to compute `span start logits` and `span end logits`). """,
+    ALBERT_START_DOCSTRING,
+)
+class TFAlbertForQuestionAnswering(TFAlbertPreTrainedModel):
+    def __init__(self, config, *inputs, **kwargs):
+        super(TFAlbertForQuestionAnswering, self).__init__(config, *inputs, **kwargs)
```
Comment (julien-c) [inline src/transformers/modeling_tf_albert.py:837]: unneeded Python 2 remnant, it's now just `super()`

Comment (Pierrci) [reply src/transformers/modeling_tf_albert.py:837]: Oh ok thank you I'm gonna change that, I was a bit lost by the different ways of doing XD

**Review by LysandreJik — APPROVED** [review]
Looks good, thanks @Pierrci !!

**Pull request merged**